src/api/app/assets/stylesheets/webui2/list-group-item-callout.scss

```diff
@@ -0,0 +1,32 @@
+.list-group-item-callout-warning {
```

Comment:
I like the colors, but I'm concerned for colour blind users. On top of this, I believe that everybody would benefit from having icons and tooltips for these icons, it's then clearer what news are (warning, success or danger)

I would also rename danger to alert and success to info.

I suggest these icons:
- Info: https://fontawesome.com/icons/info?style=solid
- Warning: https://fontawesome.com/icons/exclamation?style=solid
- Alert: https://fontawesome.com/icons/exclamation-triangle?style=solid

Reply:
I will try to add the icons there.. I was using borders so I have to change things a make space for the icons too... I will be no that subtle as now, but helpful for all the people :+1:

Reply:
I would add icons additional to the border :)

Reply:
I would add icons additional to the border :)

Reply:
I will implement it like in the Latest Updates, so an icon in the left side and then the text in 2 lines on the right